report_thesis/src/sections/background.tex

```diff
@@ -206,7 +206,72 @@ \subsubsection{Support Vector Regression (SVR)}
+$$
+where $\hat{y}_{R1}$ and $\hat{y}_{R2}$ are the mean responses for the training observations in regions $R1$ and $R2$, respectively.
+
+Once optimal splits are identified, the tree is constructed by repeating this process for each subset until a stopping criterion is met. 
+The final model consists of splits that create distinct regions, each with a predicted response value based on the mean of the observations in that region.
+
+The method ensures that each region is more homogeneous with respect to the target variable, effectively reducing impurity and making the data points within each region more similar.
+
+The predictions from the decision trees can then be improved by growing the trees sequentially, with each new tree correcting the errors of the previous ones. 
+This technique is referred to as \textit{Gradient Boosting}. 
+Each tree is small, with few terminal nodes, preventing large adjustments based on a single tree's predictions. 
+It also ensures that each tree makes small and simple error corrections, such that each step refines the model's performance more reliably.
+Finally, small trees have the property of high bias, meaning that they are unable to capture complex relationships in the data and low variance.
+By combining multiple small trees, the model can capture complex relationships while maintaining a low variance.
+
+The model is constructed iteratively by adding trees to correct the residuals of the previous trees. 
+Initially, the prediction is set as $\hat{f}_0(x) = 0$ and residuals as $r_i = y_i$ for all $i$ in the training set. 
+At each iteration $b$, predictions are updated as:
+$$
+    \hat{f}(x) = \hat{f}(x) + \lambda \hat{f}_b(x)
+$$
+where $\hat{f}_b(x)$ is the prediction of the $b$-th tree and $\lambda$ is the learning rate. Residuals, representing the errors to be corrected by the next tree, are updated as:
+$$
+    r_i = y_i - \hat{f}(x)
+$$
+Each tree is then trained on these updated residuals. This process of fitting a weak learner to predict the residuals and using its predictions to update the model is repeated for $B$ iterations, culminating in the final model:
+$$
+    \hat{f}(x) = \sum_{b=1}^{B} \lambda \hat{f}_b(x) 
+$$
+This final formulation effectively reduces errors by moving in the direction of the steepest descent as dictated by the gradient of the loss function. The commonly used squared error loss function is:
+$$
+L = \frac{1}{2}(y - \hat{y})^2
+$$
+which is continuous and differentiable, facilitating the computation of the gradient and ensuring stable optimization paths. The gradient, defined as:
+$$
+    \nabla L = y - \hat{f}(x),
+$$
+guides the model adjustments to efficiently reduce errors. 
+The learning rate $\lambda$ moderates the size of these adjustments, helping to maintain stability and prevent overfitting as the model iteratively fits to the residual errors~\cite{gradientLossFunction}.
```

Comment:
This part of the section is missing a ton of definitions for the mathematical symbols. Many symbols are undefined, and some are defined several lines after they are first introduced. Also, consider whether we event *need* all of these formulas. We decided to exclude most mathematical formulas from most of the other sections in the interest of brevity, and we only kept the most essential ones, which I don't think applies to this section.